File: truncate.js
const fs = require('fs');
const filepath = 'src/components/hbb/hbb-agent-dashboard.tsx';
const lines = fs.readFileSync(filepath, 'utf8').split('\n');
fs.writeFileSync(filepath, lines.slice(0, 567).join('\n'));
console.log('Done truncating');

File: truncate.py
import sys
import os

filename = r"C:\DEV\PWA\Airtel Champions App Web\src\components\hbb\hbb-agent-dashboard.tsx"
lines = []
with open(filename, 'r', encoding='utf-8') as f:
    lines = f.readlines()
with open(filename, 'w', encoding='utf-8') as f:
    f.writelines(lines[:568])
